File: EacademyApp-SPA/src/app/_models/student.ts
import { Course } from './course';
import { CourseStudents } from './courseStudents';

export interface Student {
    id: number;
    username: string;
    name: string;
    surname: string;
    photoURL: string;
    created: string;
    enrollmentDate: string;
    courseStudents?: CourseStudents[];
    roles?: string[];
    isInstructor?: boolean;
}

File: EacademyApp.API/Models/Student.cs
using System;
using System.Collections.Generic;

namespace EacademyApp.API.Models
{
    public class Student
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public byte[] PasswordHash { get; set; }
        public byte[] PasswordSalt { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string PhotoURL { get; set; }
        public DateTime Created { get; set; }
        public DateTime EnrollmentDate { get; set; }
        public ICollection<CourseStudent> CourseStudents { get; set; }
        public ICollection<StudentRole> StudentRoles { get; set; }
    }
}